### DISCUSSION TITLE
FileNotFoundError in jupyter notebook

### DISCUSSION THREAD

**Discussion by Kawanz** [facebookresearch/fastMRI#324] [Q&A] [answered]
Hi! I am a beginner. I am running the tutorial jupyter notebook but unable to load the h5py file. Can anyone help me out? Thanks!
file_name = 'multicoil_train/file1000167.h5'
hf = h5py.File(file_name)
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-5-4e2fd0da64cb> in <module>
      1 file_name = 'multicoil_train/file1000167.h5'
----> 2 hf = h5py.File(file_name)

**Comment by mmuckley** [comment, marked as answer]
Hello @Kawanz, you have to download the fastMRI data yourself. Then you can put that file at that path. You can download the data from https://fastmri.med.nyu.edu/.

**Reply by Kawanz** [reply]
Thank you!
